deepchem/models/torch_models/layers.py

```diff
@@ -4942,26 +4942,34 @@ def __init__(self, n_one: List[int], n_two: List[int], no_of_atoms: int,
         # Initializing the first layer (first layer has different dims than others)
         self.v.append(
             nn.Linear(8 + 3 * 4 * self.no_of_atoms, self.n_one[0], bias=True))
-        #filling the weights with 1e-9 for faster convergence
-        self.v[0].weight.data.fill_(1e-9)
-        self.v[0].bias.data.fill_(1e-9)
+        #filling the weights with 2.5e-7 for faster convergence
```

Comment:
Why are we making this change? What is the purpose of the doubling?

Reply:
Removed double datatypes as per our previous conversation. (Thought double datatypes will improve precision, but slowed down the model)